bindings/python/manifold3d.cpp

```diff
@@ -294,73 +332,35 @@ NB_MODULE(manifold3d, m) {
           "\n\n"
           ":param v: [X, Y, Z] rotation in degrees.")
       .def(
-          "rotate",
-          [](Manifold &self, float xDegrees = 0.0f, float yDegrees = 0.0f,
-             float zDegrees = 0.0f) {
-            return self.Rotate(xDegrees, yDegrees, zDegrees);
-          },
-          nb::arg("x_degrees") = 0.0f, nb::arg("y_degrees") = 0.0f,
-          nb::arg("z_degrees") = 0.0f,
-          "Applies an Euler angle rotation to the manifold, first about the X "
-          "axis, then Y, then Z, in degrees. We use degrees so that we can "
-          "minimize rounding error, and eliminate it completely for any "
-          "multiples of 90 degrees. Additionally, more efficient code paths "
-          "are used to update the manifold when the transforms only rotate by "
-          "multiples of 90 degrees. This operation can be chained. "
-          "Transforms are combined and applied lazily."
-          "\n\n"
-          ":param x: X rotation in degrees. (default 0.0).\n"
-          ":param y: Y rotation in degrees. (default 0.0).\n"
-          ":param z: Z rotation in degrees. (default 0.0).")
-      .def(
-          "warp",
-          [](Manifold &self, const std::function<Float3(Float3)> &f) {
-            return self.Warp([&f](glm::vec3 &v) {
-              Float3 fv = f(std::make_tuple(v.x, v.y, v.z));
-              v.x = std::get<0>(fv);
-              v.y = std::get<1>(fv);
-              v.z = std::get<2>(fv);
-            });
-          },
-          nb::arg("f"),
+          "warp", &Manifold::Warp, nb::arg("f"),
           "This function does not change the topology, but allows the vertices "
           "to be moved according to any arbitrary input function. It is easy "
           "to create a function that warps a geometrically valid object into "
           "one which overlaps, but that is not checked here, so it is up to "
           "the user to choose their function with discretion."
           "\n\n"
           ":param f: A function that modifies a given vertex position.")
+      .def("warp_batch", &Manifold::WarpBatch, nb::arg("f"),
+           "Same as Manifold.warp but calls `f` with a "
+           "ndarray(shape=(N,3), dtype=float) and expects an ndarray "
+           "of the same shape and type in return. The input array can be "
+           "modified and returned if desired. "
+           "\n\n"
+           ":param f: A function that modifies multiple vertex positions.")
```

Comment:
thinking about it, we may want documentation about the type signature of the function. same for `warp` above and `set_properties` below.